keyboards/boardwalk/boardwalk.h

```diff
@@ -0,0 +1,82 @@
+#ifndef BOARDWALK_H
+#define BOARDWALK_H
```

Comment:
```suggestion
```

Reply:
Updated in https://github.com/qmk/qmk_firmware/pull/5195/commits/e23d9758c802d43be8763bd7982cfec5220f1d82